CHANGELOG.md

```diff
@@ -1,13 +1,19 @@
 # @agentuity/sdk
 
-## 0.0.89
+# Changelog
+All notable changes to this project will be documented in this file.
 
-### Patch Changes
```

Comment:
can you update the other release versions to add the date to them like the other headings?